manifests/prometheus/alerts.d/exchange-rate.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- type: replace
+  path: /instance_groups/name=prometheus2/jobs/name=prometheus2/properties/prometheus/custom_rules?/-
+  value:
+    name: ExchangeRatesDrifting
+    rules:
+      - alert: ExchangeRatesDrifting_Critical
+        expr: "paas_currency_real_ratio > paas_currency_configured_ratio"
```

Comment:
It reads like this will alert on any decrease in purchasing power. I wonder if we're better off detecting anything outside a 5–10% difference, as otherwise we will see this alarm go off very often (and from temporary changes.) That's only my opinion though.

Reply:
My reasoning for this value was, if we're eating money with the rate that the ECB gives us, we're probably eating money from whatever exchange rate AWS provides, but yeah your comment makes sense the more I think about it

In your opinion would:

```
(paas_currency_real_ratio / paas_currency_configured_ratio) > 1.025
```

![Screen Shot 2019-08-06 at 17 22 23](https://user-images.githubusercontent.com/1482692/62557381-cacf3000-b86e-11e9-8917-6988a5794d70.png)

Reply:
Looks good. I'd gently nudge that we should be updating the exchange rate when it changes in our favour too.

Reply:
There is now an alert for this as well :)